src/app/pages/clients/clients.page.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ModalController } from '@ionic/angular';
+import ClientUpdateUtils from '../../utils/ClientUpdateUtils';
+import { NotificationBase } from '../../tab-preferences/notification-base';
+import { SyncService } from '../../services/sync.service';
+
+@Component({
+  selector: 'app-clients',
+  templateUrl: './clients.page.html',
+  styleUrls: ['./clients.page.scss'],
+})
+export class ClientsPage implements OnInit {
+
+  @Input() clientIdentifier: string = ""
+
+  constructor(
+    private modalCtrl: ModalController,
+    protected updateUtils: ClientUpdateUtils,
+    protected sync: SyncService,
+    public notificationBase: NotificationBase
+  ) { }
+
+  ngOnInit() {
+    this.notificationBase.lockedToggle = true
```

Comment:
We could have a unique toggle for this that is only used here instead of using `this.notificationBase.lockedToggle`. Thoughts?

Reply:
I think it's ok to reuse since there is no overlap, that way we don't have to do one property for each category